lib/miq_automation_engine/service_models/miq_ae_service_iso_datastore.rb

```diff
@@ -0,0 +1,5 @@
+module MiqAeMethodService
+  class MiqAeServiceIsoDatastore < MiqAeServiceModelBase
+    expose :iso_images, :association => true
```

Comment:
No problem exposing `ext_management_system`. I just wanted to make this PR minimal to what is exactly needed. If you want I can update this PR.